build.gradle.kts

```diff
@@ -150,6 +156,44 @@ tasks.register<Jar>("cleanShadowJar") {
     }
 }
 
+tasks.register<Jar>("shadeSourcesJar") {
+//    archiveClassifier.set("shaded-sources")
+    archiveClassifier.set("sources")
+    dependsOn("sourcesJar")
+
+    val shadingOriginPath: String = shadingOriginPackage.replace('.', '/')
+    val shadingDestinationPath: String = shadingDestinationPackage.replace('.', '/')
+    val tmpShadowedSourcesPath = "${buildDir}/tmp/shadowedSources"
```

Comment:
Any chance we get some files in this folder left from a previous build?

Reply:
Good catch! Definitely.
Yeah that is something I missed and will need to add.

Reply:
Done!